docs/docsite/rst/user_guide/intro_inventory.rst

```diff
@@ -90,6 +108,23 @@ In YAML:
           ansible_port: 5555
           ansible_host: 192.0.2.50
 
+In TOML:
+
+.. code-block:: guess
```

Comment:
toml? guessing will probably pick ini

Reply:
There is no `toml` lexer.  `guess` does indeed use `ini`, which is close enough for for me.

Reply:
https://github.com/liluo/pygments-github-lexers this should help